### DISCUSSION TITLE
Clicking a link in a readme scrolls to the top and not to the actual linked sectionFor example

### DISCUSSION THREAD

**Discussion by neuos** [community/community#101894] [Mobile]
https://github.com/awesome-selfhosted/awesome-selfhosted?tab=readme-ov-file#calendar--contacts

Scrolls to the top of the page and not to the calendar and contacts section

**Comment by ananuness** [comment]
Strange, it's going to the Calendar & contacts section here

**Reply by neuos** [reply]
I am using the mobile app v1.146.0 (767) on a Samsung Galaxy A33 running Android 14
